README.md

```diff
@@ -1,26 +1,210 @@
+* PowerShell 6+ on Windows, Linux, or macOS
+
 
 # Installing
 
-To install globally:
+We recommend installing PxGet globally from the PowerShell Gallery into the current user's scope:
+
+```powershell
+Install-Module -Name 'PxGet' -Scope CurrentUser -Repository 'PSGallery' -Force
+```
+
+If you only have the "PSGallery" repository (run `Get-PSRepository` to get a list), you can omit the `-Repository`
+parameter. If "PSGallery" repository is trusted (run `Set-PSRepository` to configure a repository's installation
+policy), you cam omit the `-Force` parameter.
+
+
+# Getting Started
+
+In your source code repository, create a "pxget.json" file in the root of your repository. It should have a `PSModules`
+property that is an array of modules that should be installed.
+
+```yaml
+{
+    "PSModules": [
+        {
+            "Name": "Whiskey",
+            "Version": "0.*"
+        },
+        {
+            "Name": "Yodel",
+            "Version": "1.*"
+        }
+    ]
+}
+```
+
+Then, open a PowerShell prompt in the same directory as the "pxget.json" file and run
 
 ```powershell
-Install-Module -Name 'PxGet'
-Import-Module -Name 'PxGet'
+pxget install
 ```
 
-To install privately:
+When PxGet is done running, there should be a "PSModules" directory in the current directory that contains all the
+private modules listed in the "pxget.json" file.
+
+
+# Adding to Builds
+
+To add PxGet to your build process, you'll need to install then run it. Run these PowerShell commands together or
+separately:
 
 ```powershell
-Save-Module -Name 'PxGet' -Path '.'
-Import-Module -Name '.\PxGet'
+Find-Module -Name 'PxGet' | Select-Object -First 1 | Install-Module -Scope CurrentUser -Force
+pxget install | Format-Table
 ```
+
+Use the `-Force` switch with `Install-Module` so the module gets installed even if the repository it is being installed
+from is untrusted.
+
+The `Select-Object -First 1` command is included in case your build server has multiple PowerShell repositories defined.
+You can omit it if there is only one, or use the `Find-Module` command's `Repository` parameter to import from a
+specific repository.
+
+The `pxget install` command returns module objects (returned by `Get-Module`). Some build systems display those as a
+list instead of a table, so we pipe the install output to `Format-Table` to make it look better in build output.
+
+
+# Configuration
+
+## Overview
+
+Each module object in the "pxget.json" file can have three properties, `Name`, `Version`, and `Path`. Only `Name` is
+mandatory, and it must be the name of the module to install. The same module may be listed multiple times, if you need
+multiple versions or need it installed in multiple directories.
+
+## Version
+
+The `Version` property is optional. If omitted, PxGet will always install the latest version. Use wildcards to pin to
+specific minor, patch, or prerelease versions of a module. PxGet assumes modules use
+(Semantic Versioning)[https://semver.org].

```

Comment:
This markdown link syntax is backwards. Swap the places of the parentheses with square brackets.